operations/mimir-mixin/dashboards/dashboard-utils.libsonnet

```diff
@@ -864,4 +883,34 @@ local utils = import 'mixin-utils/utils.libsonnet';
       replaceFields: replaceFields,
     }),
 
+  exploreContainerLogsLink(container)::
+    local exploreManifest = {
+      datasource: 'DOLLARLOKIDATASOURCE',
+      queries:
+        [
+          {
+            refId: 'A',
+            expr: '{namespace="DOLLARNAMESPACE", container=~"%s"}' % container,
+            queryType: 'range',
+            datasource: { type: 'loki', uid: 'DOLLARLOKIDATASOURCE' },
+            editorMode: 'code',
+          },
+        ],
+      range: { from: 'now-1h', to: 'now' },
+    };
+    local encodedExploreManifest = url.escapeString(std.manifestJsonEx(exploreManifest, ''));
+    // We don't want to encode the ${...} vars, since they will be rendered by Grafana.
+    local encodedExploreManifestWithVars = std.strReplace(std.strReplace(encodedExploreManifest, 'DOLLARLOKIDATASOURCE', '${lokidatasource}'), 'DOLLARNAMESPACE', '${namespace}');
+
+    local link = '%(grafana_root_path)sexplore?left=%(left_explore_urlencoded_json)s' % {
+      grafana_root_path: $._config.grafana_root_path,
+      left_explore_urlencoded_json: encodedExploreManifestWithVars,
+    };
+
+    |||
+      [Explore logs for in ${lokidatasource}](%(link)s)
```

Comment:
> Explore logs for

For...? :)

Reply:
Fixed in https://github.com/grafana/mimir/pull/3964/commits/65aae56205f57bfddb6569d2402bbcde19e19199